spec/models/admin_user_spec.rb

```diff
@@ -0,0 +1,22 @@
+describe AdminUser do
+  it 'return type should be User' do
+    testuser = AdminUser.find_or_create
+    expect(testuser).to be_a_kind_of(User)
```

Comment:
In general we don't test for types, but you can keep this test case but change it to `AdminUser` to make it more specific.